capture/runbook_test.go

```diff
@@ -17,13 +17,16 @@ func TestRunbook(t *testing.T) {
 		book string
 	}{
 		{filepath.Join(testutil.Testdata(), "book", "http.yml")},
+		{filepath.Join(testutil.Testdata(), "book", "http_multipart.yml")},
```

Comment:
https://github.com/k1LoW/runn/pull/283#issuecomment-1318150788

We think `multipart/form-data`  and `application/x-www-form-urlencoded` should be compatible as YAML formats.
So we think it is fine to leave it as it is now.

TestRunbook's problem is caused by a problem in capture/runbook.go, so I think this line can be removed.

Reply:
( I think it probably needs to parse multipart/form-data request body at capture/runbook.go. ref: https://github.com/k1LoW/runn/pull/283#discussion_r1024579919)